tests/pytests/integration/utils/test_smb.py

```diff
@@ -0,0 +1,328 @@
+"""
+Test utility methods that communicate with SMB shares.
+"""
+import getpass
+import logging
+import os
+import shutil
+import signal
+import subprocess
+import tempfile
+import time
+from pathlib import Path
```

Comment:
```suggestion
import pathlib
```

Reply:
Why import all of pathlib when only need Path, and this ```from pathlib import Path``` is used in Salt, for example: salt/utils/templates/py.
Wondering why you prefer importing all of pathlib ?

Reply:
will change to usage pathlib.Path as per conversation

Reply:
done